airflow/operators/trigger_dagrun.py

```diff
@@ -250,7 +263,7 @@ def execute(self, context: Context):
                 )
                 time.sleep(self.poke_interval)
 
-                dag_run.refresh_from_db()
+                dag_run.refresh_from_db()  # TODO here fail on internal API
```

Comment:
When adding such `TODO`, I think it would be helpful to associate it with a GitHub issue, so we'll be able to track and eventually resolve it

Reply:
I assume as the implementation will need to be re-worked for Airflow 3.0 / AIP-72, does not make sense (short term) to track this as issue. Might be a "dead horse". Changed comment(s) accordingly.